Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/NewMeterWizard.tsx

```diff
@@ -388,14 +404,27 @@ export default function NewMeterWizard(props: {IsEngineer: boolean}) {
     const secondaryHeader = React.useMemo(() => {
         if (currentStep === eventChannelsStep)
             return <p className="pull-right">
-                Number of Event Channels: {channels.reduce((p, c) => c.Trend? p : (p+1),0)}
+                Number of Event Channels: {channels.reduce((p, c) => c.Trend ? p : (p + 1), 0)}
             </p>;
         else if (currentStep === trendChannelsStep)
             return <p className="pull-right">
-                Number of Trend Channels: {channels.reduce((p, c) => c.Trend ? (p+1) : p, 0)}
+                Number of Trend Channels: {channels.reduce((p, c) => c.Trend ? (p + 1) : p, 0)}
             </p>;
         else if (currentStep === assetStep)
             return <LocationDrawings LocationID={locationInfo.ID} />
+        else if (currentStep >= additionalFieldMeterStep) {
+            return (
+                <div>
+                    <button
+                        className="btn btn-primary"
+                        style={{ position: 'absolute', top: 0, right: 0, marginRight: '80px' }}
+                        onClick={meterDetails}
+                    >
+                        Meter Details
+                    </button>
+                </div>
```

Comment:
compare styling to https://github.com/GridProtectionAlliance/SystemCenter/blob/master/Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/PropertyUI/LocationDrawings.tsx

```suggestion
                <div>
                    <button
                        className="btn btn-primary"
                        onClick={meterDetails}
                    >
                        Meter Details
                    </button>
                </div>
```